server/mcp_server_vod/src/vevod_mcp/cli.py

```diff
@@ -0,0 +1,22 @@
+ # Copyright 2025 Beijing Volcano Engine Technology Ltd.
+ # SPDX-License-Identifier: Apache-2.0
```

Comment:
As we are using MIT license in the root directory, please change the SPDX-License-Identifier to MIT instead of Apache-2.0.